tiff_2_png/tiff_2_png_converter.py

```diff
@@ -206,8 +214,13 @@ def convert_to_png(
                     ## check all images have the same data type
                     if bits_per_sample == "":
                         bits_per_sample = img.dtype
-                        global_max_pixel = np.max(img)
-                        global_min_pixel = np.min(img)
+                        if img.dtype == np.float32:
+                            global_max_pixel = 0.0
+                            global_min_pixel = 1.0
```

Comment:
Just curious why these constants need to be set for this case? Could you add a comment on why this is done when the data type is float32?

Reply:
This PR doesnt need float32 conversion. logic removed